lib/compilers/typescript-compiler.js

```diff
@@ -34,7 +35,12 @@ function getTypescriptConfig () {
   if (cachedConfig) {
     return cachedConfig
   } else {
-    const { config } = tsconfig.loadSync(process.cwd())
+    const tscPath = process.cwd()
+    let tscFile = 'tsconfig.jest.json'
+    if (!fs.existsSync(tscPath+ '/'+ tscFile)){
+        tscFile = false
+    }
+    const { config } = tsconfig.loadSync(tscPath, tscFile)
```

Comment:
We should use a jest global to get the file name, similarly to how ts-jest does it—https://github.com/kulshekhar/ts-jest/blob/master/src/preprocessor.ts#L17

You can pass a global in in the Jest config object:
```js
{
  "jest": {
    "globals": {
      "vue-jest": {
        "tsConfigFile": "my-tsconfig.json"
      }
    }
  }
}
```